zammad/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: zammad
-version: 6.0.4
+version: 6.0.5
```

New feature should raise minor version...

```suggestion
version: 6.1.0
```